Dockerfile

```diff
@@ -126,7 +126,7 @@ RUN echo "${GO_1_20_SHA256} /tmp/go1.20.linux-amd64.tar.gz" | sha256sum -c
 RUN cd /tmp && tar -C /usr/local -xvf go1.20.linux-amd64.tar.gz
 
 WORKDIR /usr/local/src/
-RUN git clone https://github.com/wealdtech/ethdo.git && cd ethdo && git fetch origin "${ETHDO_REF}" && git checkout "${ETHDO_REF}"
+RUN git clone https://github.com/ChorusOne/ethdo.git && cd ethdo && git fetch origin "${ETHDO_REF}" && git checkout "${ETHDO_REF}"
```

Comment:
Why do we need to use a fork again?

Reply:
Because upstream does not support Deneb, in fork the SSZ libraries are updated to the latest version that supports it to some degree

Reply:
we can just create an issue/send a pr

Reply:
yeah sure- you've pushed one already?